hail/python/hailtop/batch/docs/service.rst

```diff
@@ -55,29 +55,41 @@ Billing
 
 The cost for executing a job depends on the underlying machine type and how much CPU and
 memory is being requested. Currently, Batch runs all jobs on 16 core, preemptible, n1-standard
-machines with 100 GB of disk total. The costs are as follows:
+machines with 20 GB of persistent SSD boot disk and 375 GB of local SSD. The costs are as follows:
 
 - Compute cost
    = $0.01 per core per hour
 
 - Disk cost
-   .. code-block:: text
 
-       Average number of days per month = 365.25 / 12 = 30.4375
+  Average number of days per month = 365.25 / 12 = 30.4375
 
-       Cost per GB per month = $0.17
+  - Boot Disk
 
-       Cost per core per hour = $0.17 * 100 / 30.4375 / 24 / 16
+    .. code-block:: text
 
-   = $0.001454 per core per hour
+        Cost per GB per month = $0.17
+
+        Cost per core per hour = $0.17 * 20 / 30.4375 / 24 / 16
+
+  - Local SSD
+
+    .. code-block:: text
+
+        Cost per GB per month = $0.048
+
+        Cost per core per hour = $0.048 * 375 / 30.4375 / 24 / 16
+
+
+   = $0.00183 per core per hour
```

Comment:
so this seems right:
```python3
>>> 0.17 * 20 / 30.4375 / 24 / 16 + 0.048 * 375 / 30.4375 / 24 / 16
0.001830937713894593
```

But why was the old value different from this?
```python3
>>> 0.17 * 20 / 30.4375 / 24 / 16
0.0002908966461327858
```

Reply:
You need `100` instead of `20`. I think it might also be wrong. The total number was definitely correct. I may have made a previous mistake on the disk cost rate.